src/main/java/org/hmhb/organization/DefaultOrganizationService.java

```diff
@@ -146,7 +146,7 @@ public Organization save(
         }
 
         if (organization.getContactEmail() != null
-                && organization.getContactEmail().length() > publicConfig.getPhoneMaxLength()) {
+                && organization.getContactEmail().length() > publicConfig.getEmailMaxLength()) {
```

Comment:
I noticed this was wrong when looking up the config for max email (and not finding it in the org service).